client/allocrunner/taskrunner/getter/sandbox_test.go

```diff
@@ -119,13 +123,20 @@ func TestSandbox_Get_inspection(t *testing.T) {
 	testutil.RequireRoot(t)
 	logger := testlog.HCLogger(t)
 
-	// Create a temporary directory directly so the repos
-	// don't end up being found improperly
-	tdir, err := os.MkdirTemp("", "nomad-test")
-	must.NoError(t, err, must.Sprint("failed to create top level local repo directory"))
+	sandboxSetup := func() (string, *Sandbox, interfaces.EnvReplacer) {
+		testutil.RequireRoot(t)
```

Comment:
~If we've disabled filesystem isolation, shouldn't we be able to avoid requiring root?~ Oh we need it for `os.Chown`. Can we leave a comment about that? It'll make it easier to audit tests that need root in the future.